legacy_packages/sdk/src/evm/zksync/constants/addresses.ts

```diff
@@ -123,3 +160,23 @@ export function zkGetDefaultTrustedForwarders(
   //  it doesn’t need a forwarder
   return [];
 }
+
+/**
+ * Returns the native token for a given chain
+ * @param chainId - the chain id
+ * @public
+ */
+export function getZkNativeTokenByChainId(chainId: ChainId): NativeToken {
+  return (
+    NATIVE_TOKENS[chainId as number] || {
```

Comment:
if you type NATIVE_TOKENS as `Record<ChainId, ...>` then this `as` is probably not necessary?